data-tool/flows/corps_tombstone_flow.py

```diff
@@ -0,0 +1,232 @@
+    
+    for alias in snapshot_data['aliases']:
+        alias['business_id'] = business_id
+        load_data(conn, 'aliases', alias)
+    
+    for resolution in snapshot_data['resolutions']:
+        resolution['business_id'] = business_id
+        load_data(conn, 'resolutions', resolution)
+
+
+@task(name='2.2-Placeholder-Historical-Filings-Migrate-Task')
+def load_placeholder_filings(config, lear_conn: Connection, corp_num: str):
+    """Migrate placeholder historical filings."""
+    # load placeholder filings
+    # load epoch filing
+    pass
+
+
+@task(name='1-Get-Corp-Tombstone-Data-Task')
+def get_tombstone_data(config, colin_engine: Engine, corp_num: str) -> tuple[str, dict]:
+    """Get tombstone data - corp snapshot and placeholder filings."""
+    try:
+        # TODO: get filings data
+        print(f'👷 Start collecting corp snapshot and filings data for {corp_num}...')  
+        raw_data = get_snapshot_filings_data(config, colin_engine, corp_num)
+        # print(f'raw data: {raw_data}')
+        clean_data = clean_snapshot_filings_data(raw_data)
+        # print(f'clean data: {clean_data}')
+        print(f'👷 Complete collecting corp snapshot and filings data for {corp_num}!')  
+        return corp_num, clean_data
+    except Exception as e:
+        print(f'❌ Error collecting corp snapshot and filings data for {corp_num}: {repr(e)}')
+        return corp_num, None
+
+
+@task(name='2-Corp-Tombstone-Migrate-Task')
+def migrate_tombstone(config, lear_engine: Engine, corp_num: str, clean_data: dict) -> str:
+    """Migrate tombstone data - corp snapshot and placeholder filings."""
+    # TODO: update corp_processing status (succeeded & failed)
+    # TODO: determine the time to update some business values based off filing info
+    print(f'👷 Start migrating {corp_num}...')
+    with lear_engine.connect() as lear_conn:
+        transaction = lear_conn.begin()
+        try:
+            load_corp_snapshot(lear_conn, clean_data)
+            load_placeholder_filings(config, lear_conn, clean_data)
+            transaction.commit()
+        except Exception as e:
+            transaction.rollback()
+            raise e
+    print(f'✅ Complete migrating {corp_num}!')
+    return corp_num
+
+
+@flow(
+    name='Corps-Tombstone-Migrate-Flow',
+    log_prints=True,
+    persist_result=False
+)
+def tombstone_flow():
+    """Entry of tombstone pipeline"""
+    # TODO: track migration progress + error handling
+    # TODO: update unprocessed query + count query
+    try:
+        config = get_config()
+        colin_engine = colin_init(config)
+        lear_engine = lear_init(config)
+
+        total = get_unprocessed_count(config, colin_engine)
+
+        if config.BATCHES <= 0:
+            raise ValueError('BATCHES must be explicitly set to a positive integer')
+        if config.BATCH_SIZE <= 0:
+            raise ValueError('BATCH_SIZE must be explicitly set to a positive integer')
```

Comment:
if we have config for `BATCHES` and `BATCH_SIZE`, I think they should it should be specific to the tombstone flow.  i.e. the delete flow and tombstone flow should not share "batch" related config.